generators/client/templates/angular/package.json.ejs

```diff
@@ -138,7 +138,8 @@
     "write-file-webpack-plugin": "4.2.0"
   },
   "engines": {
-    "node": ">=8.9.0"
+    "node": ">=8.9.0",
+    "yarn": ">=1.3.2"
```

Comment:
As the user can choose `Yarn` or `NPM` when generating the project, I think we should add this condition:

```
"node": ">=8.9.0"<% if (clientPackageManager === 'yarn') { %>,
"yarn": ">=1.3.2"<% } %>
```

Otherwise, `Yarn` would be mandatory ?

Reply:
oh i thought yarn was mandatory. yea, i'll fix this